tasking/pulp/tasking/services/scheduler.py

```diff
@@ -10,14 +10,13 @@
 from django.db import IntegrityError
 
 from pulp.app.models.task import TaskLock, Worker
-from pulp.tasking import worker_watcher
-from pulp.tasking.celery_instance import celery as app
+from pulp.tasking.celery_app import celery as app
 from pulp.tasking.constants import TASKING_CONSTANTS as constants
-from pulp.tasking import delete_worker
+from pulp.tasking.services import worker_watcher
 
 
 # The import below is not used in this module, but it needs to be kept here. This module is the
-# first and only Pulp module to be imported by celerybeat, and by importing pulp.server.logs, it
+# first and only Pulp module to be imported by celerybeat, and by importing pulp.app.logs, it
```

Comment:
I [commented](https://pulp.plan.io/issues/2275#note-11) about this on the logging PR so these comments can be ignored.